gulpfile.js

```diff
@@ -42,6 +42,19 @@ function icons() {
     pipe(dest("./html/images/icons/dist"));
 }
 
+function attributes_icons() {
+  return src("*.svg", { cwd: "./html/images/attributes/src" }).
+    pipe(
+      svgmin({
+      plugins: [
+        { removeMetadata: true },
+        { removeTitle: true },
+      ]
+      })
+    ).
+    pipe(dest("./html/images/attributes"));
```

Comment:
Are the generated files ignored by `.gitignore`?

Reply:
good point, I'll add that.

Reply:
generated files added to .gitignore